src/uucore/src/lib/mods/line_ending.rs

```diff
@@ -0,0 +1,42 @@
+/// Aims to provide consistent newline/zero terminator handling
+use std::fmt::Display;
+
+#[repr(u8)]
+#[derive(Clone, Copy, Debug, Default, PartialEq)]
+/// Line terminator used for printing and parsing
+pub enum LineEnding {
+    #[default] Newline = b'\n',
+    Nul = 0,
+    Space = b' ',
```

Comment:
Is this one used? I don't see it being used in this patch, but maybe I'm missing something.

Reply:
Thanks for spotting this. I planed to use it at some point in the past...